### PR TITLE
Set the templates dir path for unit tests

### PR DESCRIPTION
**Pull request by gokulavasan**

### CODE REVIEW

**Comment by gokulavasan** [comment]
Will use TEMP_DIR instead of fixed "plugins"

File: cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java [hunk 1]
```diff
@@ -97,6 +97,11 @@
     public static final String MAPREDUCE_JOB_CLIENT_CONNECT_MAX_RETRIES = "mapreduce.jobclient.connect.max.retries";
 
     /**
+     * Default Template Dir.
+     */
+    public static final String DEFAULT_TEMPLATE_DIR = "plugins";
```
Comment (albertshau) [inline cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java]: this isn't the default, the default is in cdap-default.xml right?

File: cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/adapter/PluginTest.java [hunk 2]
```diff
@@ -91,6 +91,7 @@ public static void setup() throws IOException, ClassNotFoundException {
 
     cConf = CConfiguration.create();
     cConf.set(Constants.CFG_LOCAL_DATA_DIR, TMP_FOLDER.newFolder().getAbsolutePath());
+    cConf.set(Constants.AppFabric.APP_TEMPLATE_DIR, TMP_FOLDER.newFolder().getAbsolutePath());
```
Comment (albertshau) [inline cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/adapter/PluginTest.java]: this should have "plugins" too right

Comment (albertshau) [inline cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/adapter/PluginTest.java]: o nevermind i see ignore

**Comment by albertshau** [comment]
lgtm

**Pull request merged**